options.css

```diff
@@ -1,33 +1,120 @@
-h2 {
-    width: 100%;
-    margin-bottom: 0;
-    background-color: #F0F7F9; font-size: 14pt;
-    border-top: 1px solid #6B90DA;
-}
 body {
-    font-size: 12px;
+    font-size: 14px;
```

Comment:
I'm wary of the effect of these CSS changes on ChromeShack when running in Chrome.  The popup options panel for Chrome addons is very small.  Does it still look ok in Chrome?  I'm happy to support Firefox but not at the expense of Chrome.

Reply:
If these settings changes are not required for Firefox support, can they be moved into a separate PR?

Reply:
I took some liberties with styling in Firefox to make the options pane fit in better. As I mentioned in my PR comment there's a bit of work still required for the options system to be cross-compatible with Chrome.

I'll see what I can do about splitting the changes up.

Reply:
Is there a way to include both the Chrome and Firefox code paths in situations where they differ and have the JavaScript choose which code path based on the running browser?  I'd rather serve both browsers from the same git branch even if it means there is some browser-specific checking going on.

For the options pane in Chrome, there's some way to have Chrome open the options in a new tab instead of the pop-up thing.  Might be worth doing if that makes it easier to style the page the same between browsers.

Reply:
Yes. That's the goal - to serve both browsers with the same codebase.

Currently, with this PR's changes Firefox's pop-up when on the Chatty page works fine from the omni-bar. I need to thoroughly test it for regressions on Chrome, however. I will look into that idea of just using a tab instead.